tox.ini

```diff
@@ -1,12 +1,52 @@
 [tox]
-envlist = py38
+envlist = py38,lint,system,proxy
 skipsdist = true
 
 [testenv]
+deps=
+	pytest
```

Comment:
```suggestion
	pytest
	pytest-docker
	pytest-docker-db
```

Reply:
gah why? :cry:  When we have systemtest why are they not there then?